bpfman-operator/config/bpfman-deployment/daemonset.yaml

```diff
@@ -122,10 +122,6 @@ spec:
           volumeMounts:
             - name: bpfman-sock
               mountPath: /run/bpfman-sock
```

Comment:
Andrew: Why are we mounting the bpfman config in the agent?

Andre: Good catch.  I think that was a mistake during a rebase conflict fixup.  Billy had removed it as part of a recent pr, but I guess I accidentally added it back.  Removed.

Reply:
Thanks!